### PR TITLE
Move icon preloading into ui.js after css is loaded

### PR DESCRIPTION
**Pull request by humphd**

### CODE REVIEW

File: src/ui/ui.js [hunk 1]
```diff
@@ -111,12 +136,19 @@ define( [ "core/eventmanager", "./toggler", "./logo-spinner", "./context-button"
...
+        butter.loader.load(
+          [
+            {
+              type: "css",
+              url: BUTTER_CSS_FILE
+            }
+          ],
+          function(){
+            // icon preloading needs css to be loaded first
+            loadIcons( butter );
```
Comment (secretrobotron) [inline src/ui/ui.js]: Just to help future-proof functions like these, can you pass in the config instead of `butter`?

File: src/main.js [hunk 2]
```diff
@@ -674,9 +652,7 @@
         _config = Lang.defaults( userConfig, _defaultConfig );
 
         _this.project.template = _config.name;
-        resourcesDir = _config.dirs.resources || "",
-
-        loadIcons( _config.icons, resourcesDir );
+        resourcesDir = _config.dirs.resources || "";
```
Comment (secretrobotron) [inline src/main.js]: It looks like this is purposeless now.

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @secretrobotron received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/8303a1015de8a72/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @secretrobotron received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/2964d724784e14b/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/8303a1015de8a72/output.txt

Total script time: 0.29 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/2964d724784e14b/output.txt

Total script time: 0.82 mins

Server is running on http://pr333.butyr.org/templates/test.html

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @secretrobotron received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/e527ffa9188c08f/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @secretrobotron received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/68e68db4c1cd7b8/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/e527ffa9188c08f/output.txt

Total script time: 0.41 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/68e68db4c1cd7b8/output.txt

Total script time: 0.68 mins

Server is running on http://pr333.butyr.org/templates/test.html

**Pull request merged**